content/spin/quickstart.md

```diff
@@ -68,16 +68,22 @@ Download <a href="https://github.com/fermyon/spin/releases/latest" class="spin-i
 
 Unzip the binary release and place the `spin.exe` in your system path.
 
+Next, follow the steps to install the [default templates and plugins](#installing-default-templates-and-plugins).
+
 {{ blockEnd }}
 {{ blockEnd }}
 
 [See more options for installing Spin.](install)
 
 ## Install the Prerequisites
 
+> If you installed Spin with the install script, you can skip these steps, as the script installs the [default templates and plugins](install.md#installing-default-templates-and-plugins). List installed templates and plugins by running `spin templates list` and `spin plugins list`.
```

Comment:
The conditional stuff and big bunch of stuff to skip complicates the quick start a bit... I wonder if we should now position this under the Windows section because for Linux and Mac it is already done, but I doubt we can do language tabs within OS tabs.

This gets even nastier in the next section, "Install the Tools," because now for Rust and Go we still need you to install the relevant target/compiler, but for JS and Python we no longer need you to install the plugin, _unless_ you are on Windows in which case...

It would be really good to keep the quick start on rails, with no "oh now I need to skim and figure out the next step that's relevant to me."  But I am not sure what our options are.

Reply:
I think we can simply remove this. Worst case, is that people will run the commands with an update flag, and everything will be fine :-)

Reply:
I agree with removing the warning, since it is a good walk through of what the template and plugin systems are and how to engage with them

Reply:
I agree with removing the warning, since it is a good walk through of what the template and plugin systems are and how to engage with them

Reply:
Thanks Kate, I see this is removed now.